pkg/storage/internalstorage/resource_storage.go

```diff
@@ -116,14 +116,18 @@ func (s *ResourceStorage) Update(ctx context.Context, cluster string, obj runtim
 		updatedResource["deleted_at"] = sql.NullTime{Time: deletedAt.Time, Valid: true}
 	}
 
-	result := s.db.WithContext(ctx).Model(&Resource{}).Where(map[string]interface{}{
-		"cluster":   cluster,
-		"group":     s.storageGroupResource.Group,
-		"version":   s.storageVersion.Version,
-		"resource":  s.storageGroupResource.Resource,
-		"namespace": metaobj.GetNamespace(),
-		"name":      metaobj.GetName(),
-	}).Updates(updatedResource)
+	result := s.db.WithContext(ctx).
+		Model(s.model).
+		Where(map[string]interface{}{
+			"cluster":   cluster,
+			"group":     s.storageGroupResource.Group,
+			"version":   s.storageVersion.Version,
+			"resource":  s.storageGroupResource.Resource,
```

Comment:
`<group>. <version>. <version>` table to record the gvr seems to be fine, but it is possible to omit the gvr from where, which of course can be done in another pr.

Reply:
ok，I'm with you